sui/src/sui-move.rs

```diff
@@ -35,6 +45,13 @@ impl MoveCommands {
         Ok(())
     }
 
+    fn print_hex(path: &Path, is_std_framework: bool) -> Result<Vec<String>, anyhow::Error> {
```

Comment:
This function seems simple enough that it probably doesn't deserve to be a separate function